packages/datagateway-common/src/card/entityCard.component.tsx

```diff
@@ -76,9 +76,12 @@ const useCardStyles = makeStyles((theme: Theme) => {
     shadowVisible: {
       position: 'absolute',
       height: 30,
-      minWidth: '30vw',
+      minWidth: '66.5%',
```

Comment:
This specific value is enough to cover all of the text but not cover the divider to the right of the text. However, this only fully works if the window is full size. Outside of this, part of the divider is covered but the effect isn't too noticeable. Open to suggestions on how to get this perfect.

Reply:
Edit: now set this to `minWidth: '100%'`. Still open to suggestions though

Reply:
@sam-glendenning what I am thinking of at the moment is to make use of the `descriptionRef` and getting the `clientWidth` which can then be used as the `width` of the `shadowVisible` class instead of using `minWidth`. The only issue with this is that we can only apply that to the style after we get the width of the paragraph which means it might need to be rendered first (?)

Reply:
Interesting... I'd be open to a pair programming session on this so you can show me how this is done?

Reply:
@sam-glendenning sure more than happy to! :smiley:

Reply:
This [solution](https://stackoverflow.com/questions/19014250/rerender-view-on-browser-resize-with-react) online helped address this issue.